res/openvpn-gui-res-no.rc

```diff
@@ -305,27 +305,26 @@ BEGIN
                   "--connect cnn \t\t: Koble til ""cnn"" ved oppstart. (filtypen må være med)\n" \
                   "\t\t\t   Eksempel: openvpn-gui --connect office.ovpn\n" \
                   "\n" \
-                  "Parametre som vil overstyre instillinger gjort i registret:\n" \
-                  "--exe_path\t\t: Path til openvpn.exe.\n" \
-                  "--config_dir\t\t: Path til OpenVPNs konfig-katalog.\n" \
-                  "--ext_string\t\t: Filtype for konfig-filer.\n" \
-                  "--log_dir\t\t\t: Path til loggfil-katalog.\n" \
-                  "--priority_string\t\t: Prioritet på OpenVPN processer (Se install.txt for detaljer).\n" \
+                  "Parametere som vil overstyre innstillinger gjort i registeret:\n" \
```

Comment:
Just double checked http://ordbok.uib.no/perl/ordbok.cgi?OPP=parametre&bokmaal=+

It seems both A and B are valid conjugations.  Sorry about that confusion!